### DISCUSSION TITLE
Kirim gambar, dokumen dan file via localhost

### DISCUSSION THREAD

**Discussion by shahib-mirbath** [kirimwa-id/forum#86] [Tanya Jawab] [answered]
Salam, mas rio.
Saya suka dengan API api.kirimwa.id, gratis dan banyak fitur. hehehe
Semoga terus berkembang.

Setelah saya berhasil kirim pesan text, saya mencoba kirim gambar dari localhost, ternyata tidak bisa.
apa ada cara lain jika sumber gambar/file/dokumen itu dari lokal.

Terimakasih
Salam.

**Comment by rioastamal** [comment, marked as answer]
@shahib-mirbath,
Saat ini API KirimWA.id tidak menerima file dari local machine. Cara paling mudah adalah dengan melakukan upload file tersebut ke server agar bisa diakses oleh server API KirimWA.id dari internet.

Jika ingin melakukan testing file dari localhost, saya sarankan menggunakan [ngrok](https://ngrok.com/) atau [Cloudflare tunnel](https://developers.cloudflare.com/pages/how-to/preview-with-cloudflare-tunnel) agar file web server di localhost dapat diakses dari internet sehingga API KirimWA.id dapat mengaksesnya.
